test/integration/go.mod

```diff
@@ -42,6 +43,7 @@ replace (
 	k8s.io/apiserver => /sd/workspace/src/k8s.io/kubernetes/staging/src/k8s.io/apiserver
 	k8s.io/cli-runtime => /sd/workspace/src/k8s.io/kubernetes/staging/src/k8s.io/cli-runtime
 	k8s.io/client-go => /sd/workspace/src/k8s.io/kubernetes/staging/src/k8s.io/client-go
+	k8s.io/klog/v2 => /sd/workspace/src/k8s.io/kubernetes/staging/src/k8s.io/klog/v2
```

Comment:
Since klog is not available in kuberenetes 1.17 version, can you try removing this line and try